samples/web/content/apprtc/js/infobox.js

```diff
@@ -50,12 +51,14 @@ InfoBox.prototype.pushErrorMessage = function(msg) {
   this.showInfoDiv();
 };
 
-InfoBox.prototype.setCallSetupDelay = function(delay) {
-  this.callSetupDelay_ = delay;
+InfoBox.prototype.setSetupTimes = function(startTime, endTime) {
+  this.startTime_ = startTime;
+  this.endTime_ = endTime;
```

Comment:
Reading this for the first time, I kept thinking endTime_ was the end time of the call, might be good to name it startSetupTime_ and endSetupTime_